src/hotspot/cpu/riscv/vm_version_riscv.cpp

```diff
@@ -31,11 +31,17 @@
 #include "utilities/macros.hpp"
 
 const char* VM_Version::_uarch = "";
+const char* VM_Version::_vm_mode = "";
 uint32_t VM_Version::_initial_vector_length = 0;
 
 void VM_Version::initialize() {
   get_os_cpu_info();
 
+  // check if satp.mode is supported, currently supports up to SV48(RV64)/SV32(RV32)
+  if (get_satp_mode() > RISCV64_ONLY(VM_SV48) RISCV32_ONLY(VM_SV32)) {
```

Comment:
I am not sure whether it makes sense to consider SV32 here. We didn't see such an issue for RV32 zero yet.

Reply:
As RISC-V32 only has Zero support, it's safe to remove SV32. We can add it back when RV32 backend is ready.